scripts/dashboard-importer/src/dashboards/converter/converter.ts

```diff
@@ -112,10 +112,10 @@ export default class GrafanaDashboardConverter {
       },
       labels: {
         'goog-imported-via-script': '',
-        'cloud-ops-grafana-importer': '',
+        'cloud-monitoring-dashboard-importer': '',
         'goog-imported-grafana-dashboard': '',
-        [`goog-imported-grafana-revision-${this.version}`]: '',
-        ...(this.gnetId && {[`goog-imported-grafana-id-${this.gnetId}`]: ''}),
+        'goog-imported-grafana-revision': `${this.version}`,
```

Comment:
I was thinking about this, I think the concern here would be handling inconsistent typing (number vs string).  With the current approach we can reliable know it will always be a string.

Reply:
SGTM!

Reply:
Thank you!